ultratrace2/model/files/bundle.py

```diff
@@ -45,25 +45,27 @@ class FileBundleList:
         ]
     )
 
-    def __init__(self, path: str, extra_exclude_dirs: List[str] = []):
+    def __init__(self, extra_exclude_dirs: Sequence[str] = []):
```

Comment:
I think this is okay in this case, but in general you really shouldn't have mutable objects as defaults (c.f. https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/)
```suggestion
    def __init__(self, extra_exclude_dirs: Optional[Sequence[str]] = None):
```

Reply:
(superceded)